lib/Basics/UnshackledMutex.h

```diff
@@ -33,13 +33,12 @@ namespace arangodb::basics {
 // This mutex lifts this requirement and can be unlocked from another thread.
 class UnshackledMutex {
```

Comment:
do we still need a wrapper?

Reply:
Yes?
See _operationMutex usage in https://github.com/arangodb/arangodb/blob/devel/arangod/Replication2/ReplicatedLog/LogCore.cpp

Reply:
@gnusi is this question resolved?

Reply:
@joerg84 yes